app/javascript/components/Story/Story.scss

```diff
@@ -63,7 +63,11 @@
     .Story_link {
       color: #828282;
       font-size: 12px;
-      margin-left: $spacing-base;
+      display: inline;
```

Comment:
I'm not sure we need this, as it's a `<a>` element, the default display should be `inline` already.

Reply:
Oops, good point